### ISSUE TITLE
When i have interaction with notification, the app selected is pushed continuously in main screen

### ISSUE THREAD

**Issue by adidos87** [SamSprung/SamSprung-TooUI#87]
When i have interaction with notification, the app selected(whataspp, Microsoft authenticator) is pushed continuously in main screen. I've killed samsprung to resolve
[samsprung_logcat (9).txt](https://github.com/SamSprung/SamSprung-TooUI/files/8079809/samsprung_logcat.9.txt)

**Comment by AbandonedCart** [comment]
You killed the app to resolve expected functionality? If you open the app and then open your phone, it resumes.

**Comment by adidos87** [comment]
> You killed the app to resolve expected functionality? If you open the app and then open your phone, it resumes.

Maybe I don't explain well. After I opened the phone,  while I changed to other apps,  the first app was continuously pushed on top. I will describe each step. 1. Phone closed I received a notification from Microsoft authenticator,  so directly with phone closed I approved the request (I m very enthusiastic for this, thanks) 2. I open the phone and Microsoft authenticator I'd open (until here all ok) 3 . A open browser chrome an Microsoft authenticator open another time 4. I return to chrome and Microsoft authenticator automatically open , 5. And so until I've killed sumsprung.

**Comment by AbandonedCart** [comment]
Is this still an issue?

**Comment by adidos87** [comment]
Yes, today appends another time, always after approve notification in Microsoft AutenticathorIl 18 Feb 2022 18:14, TwistedUmbrellaX ***@***.***> ha scritto:
Is this still an issue?

**Comment by Adamogreco** [comment]
Works good and gets people excited
